air-quality-ui/src/services/forecast-time-service.ts

```diff
@@ -0,0 +1,42 @@
+import { DateTime, Duration } from 'luxon'
+
+export const getLatestBaseForecastTime = (from = DateTime.utc()): DateTime => {
```

Comment:
Do we need this file?

Could the API not just find whatever the previous forecast data available is and use that?

Reply:
At the moment, the API requires these parameters.
Forecast: location_type, valid_date_from, valid_date_to
Measurements summary: measurement_base_time, measurement_time_range, location_type

We could change it so that if we don't supply them we default to latest on the API side.

But I think ultimately at some point we would need a way to create base times and valid times on the front end as we will at some point want to let the user change these. Let's discuss it

Reply:
My concern is the UI trying to figure out what times are valid, as it's not going to know what actually happened on the backend, if any forecasts don't work or the cron job fails for example. 

Might not be a problem, but the alternative could be for the backend to find the closest and return that?